crawler/db/dart.py

```diff
@@ -118,6 +121,8 @@ def set_dart_well_properties(
     """
     for prop_name, prop_value in well_props.items():
         params = (plate_barcode, prop_name, prop_value, well_index)
+        # TODO: if they change state and it was picked, not perform the change
+        #
```

Comment:
We only update samples for plates that are pending, so we will not be in the situation of having picked samples

Reply:
So we can remove the TODO commend